docs/arcadia-users-group/20240730-arcadiathemer/20240730-arcadiathemer.Rmd

```diff
@@ -0,0 +1,143 @@
+  scale_fill_arcadia()
+```
+In addition to specifying which palette to use in the `scale` function, you can also reverse the colors of the scale used with `reverse=TRUE`. You can also use other `ggplot2` or `theme` specifications on top of these functions, such as moving the position of the legend or modifying the scales to remove whitespace between the axis lines and the bars:
+```{r, additional_layers}
+ggplot(data=diamonds, aes(x=cut, fill=cut)) +
+  geom_bar() +
+  theme_arcadia(x_axis_type = "categorical") +
+  scale_fill_arcadia(palette_name = "secondary", reverse = TRUE) +
+  scale_y_continuous(expand=c(0,0)) + # removes whitespace between axis and bars
+  theme(legend.position = "bottom")
+```
+In addition to reversing the order of the colors used in the palette, you can select different indices of colors from the palettes within the `scale` function with the `start` and `end` arguments:
+```{r, indices}
+ggplot(mtcars, aes(x = hp, fill = as.factor(cyl))) +
+  geom_density(alpha = 0.8, linewidth = 0) + # remove border line from filled-in density plots
+  theme_arcadia() +
+  scale_fill_arcadia(palette_name = "blue_shades", start=2, end=5) +
+  scale_y_continuous(expand=c(0,0)) +
+  scale_x_continuous(expand=c(0,0)) # remove whitespace between both axes and the plot
+```
+The `scale` functions are used to specify which palettes to use. The `gradient` functions are used to access the gradient palettes, which work in the same way as the `scale` functions using `color` or `fill`:
+```{r, gradient}
+ggplot(data = mtcars, aes(x = hp, y = mpg, color = hp)) +
+ geom_point(size=2.5) + 
+ theme_arcadia() + 
+ gradient_color_arcadia(palette_name = "lisafrank")
+```
+There are also bicolor gradients available that are useful for heatmap plots. You can also remove the background color with `background=FALSE`, which is recommended when exporting plots, which is described below. 
+```{r, heatmap}
+library(reshape2)
+
+# heatmap of correlation matrix
+data(iris)
+iris_data <- iris[, 1:4]
+cor_matrix <- cor(iris_data)
+melted_cor_matrix <- (melt(cor_matrix))
+
+ggplot(melted_cor_matrix, aes(x=Var1, y=Var2, fill=value)) +
+  geom_tile() +
+  theme_arcadia(x_axis_type = "categorical", y_axis_type = "categorical", background = FALSE) +
+  gradient_fill_arcadia(palette_name = "reds") + 
+  theme(axis.text.x = element_text(angle = 45, hjust = 1),
+        legend.position = "top", axis.line = element_blank()) +
+  labs(x = "", y = "") +
+  scale_y_discrete(expand=c(0,0)) +
+  scale_x_discrete(expand = c(0,0))
+```
+
+## Accessing palettes and specific colors
+To view all the color palette options and the individual hex codes that comprise each palette, you can view these with `show_arcadia_palettes()` or `show_arcadia_gradients()`. The gradients also show the positions for each individual color in the gradient. You can use these lists of hex codes to create custom or reordered sets of palettes.
+```{r, show_colors}
```

Comment:
The output of this cell isn't rendered properly in the Markdown; probably it's easiest to take a screenshot of the output and manually add that to the Markdown.

Reply:
I fixed this to be separate so it's clear, but this is how these are supposed to render as vectors of hex codes/positions